data/json/construction.json

```diff
@@ -1019,6 +1019,21 @@
     "pre_special": "check_support",
     "post_terrain": "t_floor"
   },
+  {
+    "type": "construction",
+    "description": "Build Thatched Roof",
+    "category": "CONSTRUCT",
+    "required_skills": [ [ "fabrication", 3 ], [ "survival", 3 ] ],
+    "time": 120,
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [
+      [ [ "2x4", 5 ], [ "stick", 10 ] ],
+      [ [ "straw_pile", 12 ], [ "pine_bough", 12 ], [ "birchbark", 12 ] ],
+      [ [ "string_36", 4 ] ]
```

Comment:
Could probably optionally use short string, or withered plants. You'd need more though.